app-modules/consent/database/factories/ConsentAgreementFactory.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Assist\Consent\Database\Factories;
+
+use Assist\Consent\Enums\ConsentAgreementType;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Assist\Consent\Models\ConsentAgreement>
+ */
+class ConsentAgreementFactory extends Factory
+{
+    public function definition(): array
+    {
+        return [
+            'type' => fake()->randomElement(ConsentAgreementType::cases()),
```

Comment:
Do you need the `->value` here at the end?

Reply:
Good catch - fixed.